src/services/etherspot.js

```diff
@@ -305,6 +305,14 @@ export class EtherspotService {
     });
   }
 
+  isValidEnsName(name: string): Promise<boolean> | boolean {
+    try {
+      return this.sdk.validateENSName({ name });
+    } catch (error) {
+      return false; // no need to report
```

Comment:
I would consider throwing here, as failed network call will be reported as failure.